### DISCUSSION TITLE
COMO OBTENER ¡GitHub Educación!

### DISCUSSION THREAD

**Discussion by Marcela-Contreras** [community/community#123465] [GitHub Education]
### Select Topic Area

Question

### Body

Hola buena tarde, quiero acceder, pero no me deja cargar los archivos pertinentes, y para tomar la foto, me sale borrosa y no se me permite, desde el día de ayer, estoy intentando pero no e podido, quisiera que me pudieran ayudar por favor, gracias

**Comment by grandmas-favorite** [comment]
Hi @Marcela-Contreras, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
